### PR TITLE
Dependabot documentation

### PR DESCRIPTION
**Pull request by petertgiles**
Based on #2152 I've tried to codify a guide to merging Dependabot PRs.  These are just my opinions 😁 so feel free to chime in.

I also created a `documentation` top level folder so feel free to comment on that too.

### CODE REVIEW

**Comment by vd1992** [comment]
Definitely a handy thing to have, it is a regular enough task to merit documentation. 
I think the major/minor change distinction is intuitive enough that people understand implicitly, in example, 2.2.X to 2.2.Y as opposed to 2.2.X to 3.00

Looking at the future thing, can you even automate Dependabot with something as nebulous as how big an update is? (Major vs Minor)

**Comment by patcon** [comment]
> can you even automate Dependabot with something as nebulous as how big an update is?

Good point! We hadn't actually validated that it was possible before, but just found these docs which make it look simple enough:
- https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/automating-dependabot-with-github-actions#enable-auto-merge-on-a-pull-request
- https://github.com/ahmadnassri/action-dependabot-auto-merge

**Comment by petertgiles** [comment]
> can you even automate Dependabot with something as nebulous as how big an update is?

That's definitely something Patrick and I discussed.  My opinion is that we should look into it once E2E testing is in place and Chromatic diffs are running again.  Perhaps we could ease into it by automating just the patch updates first and then working our way up.

**Comment by brindasasi** [comment]
Thanks for this documentation @petertgiles 
Regd, automation , my 2 cents , we could start from auto merging patch updates and still do major version changes with manual merge.
Once we are confident on our automated tests code coverage , we could make it fully automated.

File: documentation/dependabot.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Background
+We used [Dependabot](https://github.com/dependabot) to manage the dependency updates on this project.  Currently, it is set to automatically open pull requests (PRs) every Monday for updated dependencies for NPM, Composer, Docker, and GitHub Actions.  These PRs are reviewed and merged by developers on the team.  We consider this a fairly low risk activity and encourage all developers on the team to participate.
```
Comment (yonikid15) [inline documentation/dependabot.md:2]:
```suggestion
We used [Dependabot](https://github.com/dependabot) to manage the dependency updates on this project.  Currently, it is set to automatically open pull requests (PRs) every Monday for updated dependencies for NPM, Composer, Docker, and GitHub Actions. These PRs are reviewed and merged by developers on the team.  We consider this a fairly low risk activity and encourage all developers on the team to participate.
```

Comment (yonikid15) [reply documentation/dependabot.md:2]: Removed an extra space 😄

**Review by tristan-orourke — APPROVED** [review]
Everyone seems happy with this 👍

**Review by yonikid15 — CHANGES_REQUESTED** [review]

File: documentation/dependabot.md (hunk 1 above)
Comment (yonikid15) [reply documentation/dependabot.md:2]: Nvm 😆

**Pull request merged**